flink-libraries/flink-table/src/test/scala/org/apache/flink/table/utils/TableTestBase.scala

```diff
@@ -151,21 +152,31 @@ case class BatchTableTestUtil() extends TableTestUtil {
     verifyTable(tEnv.sql(query), expected)
   }
 
-  def verifyTable(resultTable: Table, expected: String): Unit = {
+  def verifyTable(resultTable: Table, expected: String, tableEnv: BatchTableEnv): Unit = {
```

Comment:
I think once we move the `from`, `to` tests to a runtime ITCase we don't need the changes to the TableTestBase anymore.